wfe/web-front-end.go

```diff
@@ -197,7 +197,7 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h wfe
 			if !methodsMap[request.Method] {
 				msg := "Method not allowed"
```

Comment:
Talked offline but the trick here is that statusCodeFromError (when it was taking an `interface{}`) was returning StatusInternalServerError because the detail was a string and not one of the core error string things. So these detail strings were never getting added to the response string.